web-app/js/portal/search/FacetDrilldownPanel.js

```diff
@@ -169,5 +170,20 @@ Portal.search.FacetDrilldownPanel = Ext.extend(Ext.tree.TreePanel, {
                 node.setText(node.attributes.value);
             }
         });
+    },
+
+    _hidePreviousDrilldowns: function() {
+        var recordCount = this.searcher.getCount();
```

Comment:
This is a hangover from a previous implementation - it should be removed along with other getCount related code